docs/CoreConfig.md

```diff
@@ -3,36 +3,33 @@ Core Configuration Files
 
 The Haraka core reads some configuration files to determine a few actions:
 
-* loglevel
```

Comment:
Why are we removing logevel from the docs - it's still valid.  It's also not being deprecated.

Reply:
We are deprecating it, in favor of `[main]level=` in log.ini.

Reply:
I don't believe the consensus on the IRC channel was to deprecate this.

It's *far* more useful to be able to do:   `````echo LOGDEBUG > /etc/haraka/config/loglevel```` than to have to edit an .ini file.

Reply:
You can still do that. The changes in the PR permit **both** methods of configuration. If the loglevel file exists, it gets priority. During runtime, the last file to get edited takes precedence.